index.html

```diff
@@ -176,7 +176,7 @@ <h4 class="about-me-title">About me</h4>
 
                 <div class="card framework-card">
```

Comment:
**Contrast ratio ("color contrast")**:

* Please ensure that the page has a minimum contrast ratio of at least 4.5:1 for normal-size text.
Please check the report [HERE](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2022-03-01-at-08-45-54-877?overview)